catalog/tests/dags/providers/provider_api_scripts/test_stocksnap.py

```diff
@@ -39,7 +39,7 @@ def test_get_media_type():
 
 
 def test_endpoint_with_initialized_page_counter():
-    expect_result = "https://stocksnap.io/api/load-photos/date/desc/0"
+    expect_result = "https://stocksnap.io/api/load-photos/date/desc/1"
```

Comment:
This is surpising... does this difference matter? Is page 0 a meaningful page that we need to retrieve, different from page 1?

Reply:
So this is actually a more semantically appropriate tests - the way the class was set up previously is that it would produce `0` if that attribute was accessed directly after construction, but when it was _actually_ called it would be after the first `get_next_query_parameters`, which would always increment the value and set it to `1`. We want page `1` and need to start there, this change just more appropriately reflects that as the starting page (rather than relying on the expected ingestion flow which would increment it before actually calling the endpoint).

Reply:
Great!